portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,83 @@
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<div id="content">
+      <h1><p3 style="color:white">Michael Peters's Portfolio </p3></h1>
+      <center><img src="/images/IMG_3549.jpg" width="120" height="190"></center>
+      
+      
+      
+
+<div id="greeting-container"></div>
     </div>
+    <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <p4>About Me</p4>
+    </div id="Me">
+    <p1 style="color:white">My name is Michael Peters. I was born and raised in Southern California. 
+          I was raised alongside my triplet siblings, me included, and my older brother.
+        What got me into the major of Computer Information Technology was the fact that 
+        Since I was kid, i was told that going into the medical field would gurantee success.
+        However, it wasn't my main interest. It was technology that would captrue my attention.
+        Whether it was gaming or helping my family with their devices by troubleshooting and finding
+        the issue, it was always my niche.
+    </p1>
+
+    <div>
+        <p1 style="color:white"> I am now attending the California State University of Northridge and am pursuing a
+        degree in Computer Information Technology and a minor in Geographic Information Science.
+        In the future with my degree, I hope to become a Data Scientist. Another occupation that 
+        I am also highly interested in becoming is a Cyber Security Analyst.
+        </p1>
+    </div>
+
+    
+    <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <p4>My LinkedIn and GitHub</p4>
+    </div>
+    <div><p1 style="color:white"> https://github.com/mike39608/Coding-Portfolio </p1></div>
+    <div><p1 style="color:white"> https://www.linkedin.com/in/michael-peters-4686b1227/ </p1></div>
+    
+        <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <p4>My Interests</p4>
+        </div>
+        <p1 style="color:white"> When it comes to what I like, I love to play sports and video games.
+            As well as watching shows, movies and anime. Here are some I am a big fan of.
+            I am currently learning to cook becuase the hobby had always interested me. watching
+            the food channnel and social media accounts propelled my interest into the world of
+            learning to cook foods. 
+            Otherhobbies and skills I would love to learn and practice woudl be learning a new language
+            such as spanish or playing the piano.
+        </p1>
+        <div><img src="/images/under night.jpg" width="400" height="300"><img src="/images/fire force.jpg" width="400" height="300"></div>
+        
+       
+    
+        <div class="w3-panel w3-light-gray w3-round-xlarge">
+        <p4>My projects</p4>
+        </div>
+        <p1 style="color:white"> I have created projects such as a Binary Tree and 
```

Comment:
I'd insert a new line between `<p1 style="color:white">` and the rest of the paragraph to make it easier to parse visually.